examples/custom_action/example_server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+connexion >= 2.7.0
```

Comment:
I saw 2.0.0 in setup.py

Reply:
No, the other 2.0 should be updated to 2.7s.

Reply:
Done